psydac/linalg/solvers.py

```diff
@@ -365,13 +363,8 @@ def solve(self, b, out=None):
         Returns
         -------
         x : psydac.linalg.basic.Vector
-            Converged solution.
-
-        info : dict
-            Dictionary containing convergence information:
-            - 'niter'    = (int) number of iterations
-            - 'success'  = (boolean) whether convergence criteria have been met
-            - 'res_norm' = (float) 2-norm of residual vector r = A*x - b.
+            Numerical solution of the linear system. To check the convergence of the solver,
+            inspect InverseLinearOperator._info.
```

Comment:
Actually we should not read a private attribute like `_info` unless strictly needed. The `InverseLinearOperator` class has a public method which I think is called `get_info()`: we should point the user to it.